File: ivf-client/src/app/core/constants/lab.constants.ts
// Lab Schedule Constants
export const ScheduleTypes = {
    RETRIEVAL: 'retrieval' as const,
    TRANSFER: 'transfer' as const,
    REPORT: 'report' as const
};

export const ScheduleStatuses = {
    PENDING: 'pending' as const,
    DONE: 'done' as const
};

// Type for schedule type
export type ScheduleType = typeof ScheduleTypes[keyof typeof ScheduleTypes];

// Type for schedule status  
export type ScheduleStatus = typeof ScheduleStatuses[keyof typeof ScheduleStatuses];

File: src/IVF.Domain/Constants/LabConstants.cs
namespace IVF.Domain.Constants;

/// <summary>
/// Constants for schedule types used in Lab operations
/// </summary>
public static class ScheduleTypes
{
    public const string Retrieval = "retrieval";
    public const string Transfer = "transfer";
    public const string Report = "report";
}

/// <summary>
/// Constants for schedule statuses
/// </summary>
public static class ScheduleStatuses
{
    public const string Pending = "pending";
    public const string Done = "done";
}

/// <summary>
/// Constants for appointment notes
/// </summary>
public static class AppointmentNotes
{
    public const string EmbryoReport = "Báo phôi";
}
